pdr_backend/aimodel/aimodel_data_factory.py

```diff
@@ -120,6 +119,9 @@ def create_xy(
         assert X.shape[1] == ss.n
         assert isinstance(x_df, pd.DataFrame)
 
+        assert "timestamp" not in x_df.columns
```

Comment:
I'm assuming no timestamp because now it's the pd index

Reply:
Yes